components/form/demo/register.tsx

```diff
@@ -176,7 +176,7 @@ const App: React.FC = () => {
               if (!value || getFieldValue('password') === value) {
                 return Promise.resolve();
               }
-              return Promise.reject(new Error('The two passwords that you entered do not match!'));
+              return Promise.reject(new Error('The new passwords that you entered do not match!'));
```

Comment:
`The new passwords` or `The new password`?

Reply:
`The new password`